image/laboratory/opt/laboratory/configure_network.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+# Edit the network.js file to change the network used
+sed -i "s@https://horizon-testnet.stellar.org@$HORIZON_ENDPOINT_TESTNET@g" /opt/laboratory/src/constants/network.js
+sed -i "s@Networks.TESTNET@'$NETWORK_PASSPHRASE_TESTNET'@g" /opt/laboratory/src/constants/network.js
+
+# Edit the accountCreator.js file to change the friendbot service used
+sed -i "s@https://friendbot.stellar.org@$FRIENDBOT_ENDPOINT@g" /opt/laboratory/src/actions/accountCreator.js
+
+./node_modules/.bin/gulp develop
```

Comment:
why do you need gulp here? Usually it's for development

Reply:
I just followed the instructions from their README
https://github.com/stellar/laboratory/tree/master

```bash
npm install
./node_modules/.bin/gulp develop
```

Reply:
exactly, it's for development. Gulp constantly monitors your directories and reloads the app if anything is changed and does other things. You should run `gulp build` and it will create a `dist/app.js` file, which is a minified distro for production. Usually you run it with Node afterwards: `node dist/app.js`. The app should be much faster.